CONTRIBUTING.rst

```diff
@@ -0,0 +1,160 @@
+`tutorial <http://help.github.com/fork-a-repo/>`_ on how to set one up. The rest of this
+example assumes you have set up the "upstream" repository as cyclus/core. Note that git
+refers to your fork as "origin".
+
+First, let's make our "work" branch:
+::
+    .../cyclus_dir/$ git branch work
+    .../cyclus_dir/$ git push origin work
+
+We now have the following situation: there exists the "blessed" copy of the Master and
+Develop branches, there exists your fork's copy of the Master, Develop, and Work branches,
+*AND* there exists your *local* copy of the Master, Develop, and Work branches. It is
+important now to note that you may wish to work from home or the office. If you keep your
+fork's branches up to date (i.e., "push" your changes before you leave), only your *local*
+copies of your branches may be different when you next sit down at the other location.
+
+Workflow: The Beginning
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Now, for the workflow! This is by no means the only way to perform this type of
+workflow, but I assume that you wish to handle conflicts as often as possible
+(so as to keep their total number small). Let us imagine that you have been at
+work, finished, and successfully pushed your changes to your *Origin*
+repository. You are now at home and want to continue working a bit. To begin,
+let's update our *home's local branches*.  ::
+
+    .../cyclus_dir/$ git checkout develop
+    .../cyclus_dir/$ git pull upstream develop
+    .../cyclus_dir/$ git push origin develop
+
+    .../cyclus_dir/$ git checkout work
+    .../cyclus_dir/$ git pull origin work
+    .../cyclus_dir/$ git rebase develop
+    .../cyclus_dir/$ git push origin work
+
+Perhaps a little explanation is required. We first want to make sure that this new local copy of
+the develop branch is up-to-date with respect to the remote origin's branch and remote upstream's
+branch. If there was a change from the remote upstream's branch, we want to push that to origin.
+We then follow the same process to update the work branch, except:
+
+#. we don't need to worry about the *upstream* repo because it doesn't have a work branch, and
+#. we want to incorporate any changes which may have been introduced in the develop branch update.
+
+Workflow: The End
+^^^^^^^^^^^^^^^^^
+
+As time passes, you make some changes to files, and you commit those changes (to your *local work
+branch*). Eventually (hopefully) you come to a stopping point where you have finished your project
+on your work branch *AND* it compiles *AND* it runs input files correctly *AND* it passes all tests!
+Perhaps you have found Nirvana. In any case, you've performed the final commit to your work branch,
+so it's time to make a pull request online and wait for our developer friends to
+review and accept it.
+
+Sometimes, your pull request will be closed by the reviewer until further
+changes are made to appease the reviewer's concerns. This may be frustrating,
```

Comment:
OK, well if you just moved it that is fine. My disagreements were probably more philosophical.